src/System.Management.Automation/commands/utility/FormatAndOutput/common/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -717,7 +717,7 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
         private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Automation_ErrorRecord()
         {
             yield return new FormatViewDefinition("ErrorInstance",
-                CustomControl.Create(outOfBand: true)
+                CustomControl.Create(outOfBand: true, width: Int32.MaxValue)
                     .StartEntry()
```

Comment:
Thanks - I tried your code and agree that it seems to work in a reasonable way. I'm still concerned though :)